extensions/sql-migration/src/dialog/createSqlMigrationService/createSqlMigrationServiceDialog.ts

```diff
@@ -553,7 +554,8 @@ export class CreateSqlMigrationServiceDialog {
 		});
 
 		this._copyKey2Button = this._view.modelBuilder.button().withProps({
-			iconPath: IconPathHelper.copy
+			iconPath: IconPathHelper.copy,
+			ariaLabel: constants.COPY_KEY,
```

Comment:
I'd suggest having separate labels for key1 vs key2 - otherwise users might get confused as to which one they're copying. 

(same comment for all the other buttons in this PR)